app.rb

```diff
@@ -1,9 +1,28 @@
 require 'sinatra/base'
+require_relative './lib/peep'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  enable :sessions, :method_override
+
+  get '/messageboard' do
```

Comment:
It might be worth having an index route as well as that's where users will land by default on a website. You could connect it to  `/messageboard` using a redirect if that's where you want users to end up.